nixos/modules/services/monitoring/prometheus/exporters/unbound.nix

```diff
@@ -0,0 +1,72 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.unbound;
+in
+{
+  port = 9167;
+  extraOpts = {
+    fetchType = mkOption {
+      type = types.enum [ "shm" "tcp" "uds" ];
+      default = "shm";
+      description = ''
+        Which methods the exporter uses to get the information from unbound.
+      '';
+    };
+
+    listenAddress = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      description = ''
+        Address under which the exporter exposes the metrics.
+      '';
+    };
+
+    telemetryPath = mkOption {
+      type = types.str;
+      default = "/metrics";
+      description = ''
+        Path under which to expose metrics.
+      '';
+    };
+
+    socketPath = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "/run/unbound/unbound.socket";
+      description = ''
+        Path to the unbound socket
+      '';
+    };
+
+    extraParams = mkOption {
```

Comment:
There's already `extraFlags` added to each prometheus exporter module. All already existing options are documented under https://nixos.org/manual/nixos/stable/index.html#module-services-prometheus-exporters-new-exporter